src/main/java/io/antmedia/streamsource/StreamFetcher.java

```diff
@@ -153,6 +153,9 @@ public Result prepareInput(AVFormatContext inputFormatContext) {
 		String timeoutStr = String.valueOf(this.timeout);
 		av_dict_set(optionsDictionary, "stimeout", timeoutStr, 0);
 
+		String analyzeDuration = String.valueOf(appSettings.getMaxAnalyzeDurationMS());
+		av_dict_set(optionsDictionary, "analyzeduration", analyzeDuration, 0);
+
```

Comment:
Have you checked the unit? It says it is microseconds not milliseconds
https://ffmpeg.org/ffmpeg-formats.html